migrations/20170306233934-timestamp-with-tz.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(`
+ALTER TABLE events ALTER COLUMN created_at TYPE TIMESTAMP WITH TIME ZONE;
```

Comment:
How is timezone not default? So weird (timestamps should always be epoch (milli)seconds IMO)